calcom/cal.com
# Removing headlessui used in menus

## What does this PR do?

Recreating PR #2114, now it does not depend on any other branch.

Fixes #2073 

## Type of change


## How should this be tested?

 - Go to Event type page in your mobile phone and see the mobile-specific dropdown to be Radix and have UI/UX consistency to the rest of menus
- Go to Bookings page in your mobile phone and see the mobile-specific dropdown for a booking to be Radix and have UI/UX consistency to the rest of menus
